perun/postprocess/regressogram/run.py

```diff
@@ -0,0 +1,76 @@
+"""
+Postprocessor module with non-parametric analysis using the regressogram method.
+"""
+import click
+
+import perun.logic.runner as runner
+import perun.postprocess.regressogram.methods as methods
+from perun.postprocess.regression_analysis.data_provider import data_provider_mapper
```

Comment:
Fix: again use absolute imports combined with as. `from` can be used e.g. for structures or constants, but not for functions.